test/contrib/homeplugsg.uts

```diff
@@ -0,0 +1,28 @@
+% Regression tests for Scapy
+
++Syntax check
+= Import the homepluggq layer
+
+from scapy.contrib.homeplugsg import *
+import binascii
+#from scapy.all import 
+
+# HomePlugSG
+
+############
+############
++ Basic tests
+
+* Those test are here mainly to check nothing has been broken
+
+= Building packets packet
+~ basic HomePlugAV VS_UART_CMD_REQ VS_UART_CMD_CNF
+
+HomePlugAV()/VS_UART_CMD_REQ()
+HomePlugAV()/VS_UART_CMD_CNF()
```

Comment:
This isn't a useful test.
Good tests usually are:
- a few build tests:
```python
assert raw(IP(dst="127.0.0.1", src="127.0.0.1")) == b"\...."
assert raw(Ether()) == b"\...."
...
```
- a few dissection tests:
```python
pkt = Ether(b"....")
assert pkt.dst = ...
...
pkt = IP(...)
assert pkt.dst = ...
....
```
```

Ideally you would have one per layer you created. This also applies elsewhere

Reply:
Erased + some more tests have been introduced for the HPGP protocol